layout/dir.proj

```diff
@@ -13,7 +13,7 @@
     <TargetFrameworkMonikers Include="netstandard1.5" />
     <TargetFrameworkMonikers Include="netstandard1.6" />
     <TargetFrameworkMonikers Include="netstandard1.7" />
-    <TargetFrameworkMonikers Include="netcoreapp1.0">
```

Comment:
I had netcoreapp1.0 but the way I was generating it wasn't actually correctly representing it because it was still pulling the latest runtime package which showed that it had a lot more APIs then it really did. Some later point when I can point this project to a specific package I may generate it or I will generate it manually as it isn't changing any longer.